### PR TITLE
Implement sharp ratio

### PR DESCRIPTION
**Pull request by keisuke-umezawa**

### CODE REVIEW

File: src/backlight/metrics/pl.py [hunk 1]
```diff
@@ -20,6 +22,20 @@ def _divide(a: float, b: float) -> float:
     return a / b if b != 0.0 else 0.0
 
 
+def calc_sharp(
+    positions: Positions, principal: float, freq: pd.Timedelta = pd.Timedelta("60min")
```
Comment (m3at) [inline src/backlight/metrics/pl.py]: Is "60min" a standard frequency for computing sharpe? I think it's application specific, if so let's not put a default

Comment (keisuke-umezawa) [reply src/backlight/metrics/pl.py]: Okay! Thanks.

**Comment by keisuke-umezawa** [comment]
I will fix it lager if there are any comments.

**Comment by m3at** [comment]
Yes please correct the name as stated above ⬆️😁

**Pull request merged**